turbinia/client.py

```diff
@@ -1258,7 +1263,12 @@ def __init__(self, jobs_denylist=None, jobs_allowlist=None):
     # Check for valid dependencies/directories.
     dependencies = config.ParseDependencies()
     if config.DOCKER_ENABLED:
-      check_docker_dependencies(dependencies)
+      try:
+        check_docker_dependencies(dependencies)
+      except Exception as e:
+        log.warning(
+            "DOCKER_ENABLED=True but error checking docker daemon: {0:s}"
```

Comment:
Tiny nit, but just to be a little more clear/verbose, maybe we could say something similar to `DOCKER_ENABLED=True is set in the config, but there is an error checking for the docker daemon`?